graviti/schema/package.py

```diff
@@ -12,9 +12,10 @@
 from typing import TYPE_CHECKING, Any, Dict, Iterator, List, Mapping, Optional, Tuple, Type, TypeVar
 
 import yaml
-from tensorbay.utility import ReprMixin, ReprType, UserMapping
+from tensorbay.utility import UserMapping
 
-from graviti.utility import AttrDict
+from graviti.utility.attr import AttrDict
+from graviti.utility.repr import ReprMixin, ReprType
```

Comment:
`from graviti.utility import AttrDict, ReprMixin, ReprType`?
Imports from another module.

Reply:
But “from graviti.utility import AttrDict, ReprMixin, ReprType” cannot pass Build Docs check

Reply:
This is not expected, @graczhual please help on this.

Reply:
Because the circular import. Should we accept the current code, or adjust the file structure in another pr?